lib/SIL/SILPrinter.cpp

```diff
@@ -922,6 +921,12 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     case SILNodeKind::SILUndef:
       printSILUndef(cast<SILUndef>(node));
       return;
+
+#define MULTIPLE_VALUE_INST_RESULT(ID, PARENT) case SILNodeKind::ID:
```

Comment:
Pattern in this file is to put the case on the following line.

Reply:
Again, this was git-clang-format I think.